integrations/sonarqube/client.py

```diff
@@ -113,46 +111,38 @@ async def send_api_request(
             )
             raise
 
-    async def send_paginated_api_request(
+    async def _handle_paginated_request(
```

Comment:
let's rename to _send_paginated_request() so it's consistent with what we have in other integration